filebeat/crawler/prospector_log.go

```diff
@@ -228,6 +228,8 @@ func (p *ProspectorLog) checkExistingFile(h *harvester.Harvester, newFile *input
 
 			// Start a new harvester on the path
 			h.Start()
+			p.Prospector.registrar.Persist <- h.GetState()
```

Comment:
means it could happen that an old state is reported? Worst case: few lines are sent again?